src/prefect/server/api/task_runs.py

```diff
@@ -57,7 +77,18 @@ async def create_task_run(
 
     if model.created >= now:
         response.status_code = status.HTTP_201_CREATED
-    return model
+
+    new_task_run: schemas.core.TaskRun = schemas.core.TaskRun.from_orm(model)
+
+    # Place autonomously scheduled task runs onto a notification queue for the websocket
```

Comment:
This feels like the heart of the matter, in some ways, and whatever we discover while exploring these waters should be useful for e.g. replacing workers polling work queues with websocket subscriptions. But other than saying I'm intrigued I should probably pass over this in silence for the moment.

Reply:
Yes very much, and I think we can grow this functionality out to support objects in other states for other subscriptions.  In Prefect Server, we should be able to handle a fairly significant number of in-flight objects in memory.  For Prefect Cloud we'll need more of an external message broker, but the same concept applies.